docs/swagger.json

```diff
@@ -0,0 +1 @@
+{"openapi":"3.0.0","info":{"title":"CLARK Gateway","version":"2.26.6","description":"The API gateway for all of the CLARK microservices","license":{"name":"ISC","url":"https://www.isc.org/licenses/"},"contact":{"name":"SecurEd Inc","url":"http://secured.team/","email":"skaza@towson.edu"},"x-logo":{"url":"http://secured.team/assets/images/logo.png","altText":"CLARK Logo"}},"servers":[{"url":"http://localhost:3001","description":"Development"},{"url":"https://api-gateway.clark.center","description":"Production"}],"paths":{},"components":{},"tags":[]}
```

Comment:
Could we format this json object?

Reply:
It gets autogenerated every time the API runs so it wouldn't help.

Reply:
Okay cool, sounds good.